README.md

```diff
@@ -8,8 +8,8 @@ See [RubyDoc.info][rubydoc] for the API docs of the latest published gem.
 
 Install with:
 
-```
-$ gem install redis
```

Comment:
This is what Bundelr generate when running `bundle gem`, I don't see why we'd remove it.

Reply:
@byroot 

Whenever you copy a command directly from GitHub and paste it into the terminal, it's terribly annoying to have to delete the `$` sign each time. Personally, I find the dollar sign useless here, as it only wastes time to remove it and doesn't represent anything

<img width="877" alt="Screenshot 2024-04-08 at 17 21 22" src="https://github.com/redis/redis-rb/assets/91337563/9d6df052-2132-42f8-b489-e41d80354781">


<img width="269" alt="Screenshot 2024-04-08 at 17 23 13" src="https://github.com/redis/redis-rb/assets/91337563/30dcdc43-58b1-4cbb-a03c-006062bc23db">